processing/missing_value_format.py

```diff
@@ -71,6 +74,139 @@ def check_missing_values_col(self, fname, header,
+                invalid_data_row = \
+                    self.data_util.check_invalid_data_row(data_row=tokens[0])
+                if invalid_data_row:
+                    # assume units line or some other junk
+                    drop_line_count += 1
+                    continue
+                fixed_tokens = []
+                for i, t in enumerate(tokens):
+                    t = t.strip()
+                    invalid_missing_value_format, msg = \
+                        self.data_util.check_invalid_missing_value_format(t,
+                                                                        check_types=[
+                                                                            'common_value',
+                                                                            '69_value',
+                                                                            'char_value',
+                                                                            'imaginary_value',
+                                                                            'factorial_value',
+                                                                            'scientific_value'
+                                                                        ])
+                    if invalid_missing_value_format:
+                        check_log.error('{msg} on line {li} '
+                                        'column {header} [{t}]'.format(
+                                        msg=msg,
+                                        li=ln_num+1,
+                                        header=headers[i],
+                                        t=t))
+
+    def _tokenize(self, line):
+        return line.split(',')
+
+    def _strip_quotes(self, tokens):
+        """
+
+        :param tokens: list of variable names
+        :return: list of variable names with quotes removed
+                 boolean, True if quotes were removed
+        """
+        return self._strip_character(tokens, character='"')
+
+    def _strip_whitespace(self, tokens):
+        """
+
+        :param tokens: list of variable names
+        :return: list of variable names with whitespaces removed
+                 boolean, True if whitespaces were removed
+        """
+        return self._strip_character(tokens, character=string.whitespace)
+
+    def _strip_character(self, tokens, character):
+        """ Takes in a line of values and remove whitespaces and quotes
+        from the beginning or end of values if the characters exist.
+
+        Returns a list of tokens and a boolean value of True if whitespace
+        or quotes are removed
+        """
+        sum_token_len = sum((len(t) for t in tokens))
+
+        tokens = [t.strip(character) for t in tokens]
+        sum_no_character_token_len = sum((len(t) for t in tokens))
+
+        if (sum_token_len - sum_no_character_token_len) > 0:
+            return tokens, True
+        return tokens, False
+
```

Comment:
Move these to the utils file and add an issue ticket to refactor file_fixer module to use the functions from the utils.

Reply:
Refer to issue #77